qiime/metadata.py

```diff
@@ -14,12 +16,30 @@ def __init__(self, dataframe):
     def __repr__(self):
         return self._dataframe.to_json()
 
+    @staticmethod
+    def load(path):
+        try:
+            df = pd.read_csv(path, sep='\t', index_col=0, dtype=object)
```

Comment:
@eldeveloper, @josenavas @antgonza, wanted to draw attention here in the event any of the pandas + qiita experiences are relevant as I know there were a few unexpected issues encountered.